lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -320,68 +417,59 @@ def draw_pane(self, renderer):
 
         renderer.close_group('pane3d')
 
-    @artist.allow_rasterization
-    def draw(self, renderer):
-        self.label._transform = self.axes.transData
-        renderer.open_group("axis3d", gid=self.get_gid())
+    def _axmask(self):
+        info = self._axinfo
+        index = info["i"]
+        axmask = [True, True, True]
+        axmask[index] = False
+        return axmask
```

Comment:
I'd suggest either passing `index` as an argument or just stack the calls. I added the explicit variables to reuse the variables, but no reuse here.